easyconfigs/m/modkit/modkit-0.3.1rc1-GCCcore-12.2.0.eb

```diff
@@ -0,0 +1,627 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'Cargo'
+
+name = 'modkit'
+version = '0.3.1rc1'
+local_rustver = '1.75.0'
+versionsuffix = '-Rust-' + local_rustver
+
+homepage = 'https://github.com/nanoporetech/modkit'
+description = """
+A bioinformatics tool for working with modified bases from Oxford Nanopore.
+Specifically for converting modBAM to bedMethyl files using best practices,
+but also manipulating modBAM files and generating summary statistics.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+crates = [
```

Comment:
For consistency with other Cargo configs maybe move the `crates` list below the other deps?